nix/mayastor-overlay.nix

```diff
@@ -3,11 +3,13 @@ self: super: {
   nvmet-cli = super.callPackage ./pkgs/nvmet-cli { };
   libspdk = (super.callPackage ./pkgs/libspdk { }).release;
   libspdk-dev = (super.callPackage ./pkgs/libspdk { }).debug;
-  mayastor = (super.callPackage ./pkgs/mayastor { }).release;
-  mayastor-dev = (super.callPackage ./pkgs/mayastor { }).debug;
-  mayastor-adhoc = (super.callPackage ./pkgs/mayastor { }).adhoc;
+  mayastor-release = (super.callPackage ./pkgs/mayastor { }).release;
+  mayastor-debug = (super.callPackage ./pkgs/mayastor { }).debug;
+  mayastor-ad-hoc = (super.callPackage ./pkgs/mayastor { }).adhoc;
   moac = (import ./../csi/moac { pkgs = super; }).package;
   images = super.callPackage ./pkgs/images { };
+  services = super.callPackage ./pkgs/images/services { };
```

Comment:
It's unclear to me what the difference between `images` and `services` is? Are the services not images? Do the images (like `mayastor` -- which is a daemon/service) not operate as services?

Reply:
The distinction was meant to be between the new control plane services and the existing images, but maybe that is confusing. I'll keep them with the other images and then I can remove this.

Reply:
I have removed the `services` definition now. Only `images` remain. Apologies for the confusion.